src/features/metrics/components/MetricsList.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import Link from "next/link";
+import { useCurrentDomain } from "~/features/rounds/hooks/useRound";
+import { Metric } from "~/features/metrics/types";
+import { Card } from "~/components/ui/Card";
+
+export function MetricsList({
+  metrics,
+  isPending,
+}: {
+  metrics?: Metric[];
+  isPending: boolean;
+}) {
+  const domain = useCurrentDomain();
+
+  return (
+    <section className="space-y-4">
+      {isPending
+        ? Array(5)
+            .fill(0)
+            .map((_, i) => <MetricCard key={i} isLoading />)
+        : metrics?.map((metric) => (
+            <MetricCard
+              key={metric.id}
+              href={`/${domain}/metrics/${metric?.id}`}
+              metric={metric}
+            />
+          ))}
+    </section>
+  );
+}
+
+function MetricCard({
+  metric,
+  isLoading,
+  href = "",
+}: {
+  isLoading?: boolean;
+  metric?: Metric;
+  href?: string;
+}) {
+  const domain = useCurrentDomain();
+
+  return (
+    <Card type="metrics">
+      <div className="flex gap-8">
+        <div className="flex-1 space-y-4">
+          {isLoading ? (
+            <>
+              <Skeleton className="h-6 w-48" />
+              <Skeleton className="h-12" />
+              <Skeleton className="h-6 w-32" />
```

Comment:
There is a skeleton component in components/ui. See if you can use that one instead. Otherwise lets keep it as is here.

Reply:
done, using it now